src/App.js

```diff
@@ -1,36 +1,36 @@
-import { useEffect, useState } from "react";
-import { FiSearch } from "react-icons/fi";
-import { IMaskInput } from "react-imask";
 import "./styles.css";
-import api from "./services/api";
 import ReactGA from "react-ga4";
+import api from "./services/api";
+import { IMaskInput } from "react-imask";
+import { FiSearch } from "react-icons/fi";
+import { useEffect, useState } from "react";
 
 function App() {
-  const [input, setInput] = useState("");
   const [cep, setCep] = useState({});
-  const [errorMessage, setErrorMessage] = useState("");
+  const [input, setInput] = useState("");
   const [showError, setShowError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
 
-  function handleKeyPress(event) {
+  useEffect(() => {
+    if (showError) {
+      setTimeout(() => {
+        setShowError(false);
+      }, [3000]);
+    }
+  }, [showError]);
+
+  function handleKeyDown(event) {
     ReactGA.event({
       category: "Buscar CEP",
       action: "onKeyPress",
```

Comment:
nome da action deve seguir o padrão de nomenclatura

Reply:
Changed action name according to function